code/modules/research/designs/weapon.dm

```diff
@@ -83,17 +83,6 @@
 	build_path = /obj/item/bluespace_harpoon
 	category = "Bluespace Telecoms"
 
-/datum/design/research/item/clothing/soteriaarmorchest
-	name = "'Mark II' environmental protection suit"
```

Comment:
~~Please stop nerfing Soteria into the ground(which is already has happened via SOPs), and~~ replace the removed content with something else, or to the end of the year there will be nothing to research at all.
Especially with the fact that this armor was more-or-less fine resource-wise.

Reply:
And just to point our you already removed an entire research block in https://github.com/sojourn-13/sojourn-station/pull/4617

Reply:
@ThePainkiller  This also applies to you^.

Reply:
Chem dispensers are irrelevant/out of scope of this PR
**Soteria can't produce any resources, and can't mine via SOP** A plain lie, you can go to maintenence, to space to even just growing plants (the worst and least viable option). Still doesn't change anything on the fact for just quarter of a price you get little below half of what one of the most expensive armors in game can be. It's going away like it or not, you have laserproof armor which is now way better so enjoy that, but soteria really isn't going to be better at armor making than the literal blacksmith faction.

Reply:
@ActuallyALovelyGrave I'm no longer part of the Discord. Or the server in general if you haven't noticed. I just code stuff for friends that ask me to code stuff for them. The server can keep going in the dumpster fire direction it's going to for all I care, I just type funny words and push PR's on other's behalf, since they can't code, or are too busy to.
I just agree with your points and I hate to see the shitty, stock, metal armor being deleted for no reason as it was AT LEAST a non-mining-materials-required printable shit armor for when stuff happens, but that's just the opinion of someone who wasn't taken into account despite +100 PR's merged and a history of three years worth of contributions to the codebase and server buildup, a trend that I see happens a lot with even people with a much larger contributing history than mine, but this ain't the place for it.
The "Soteria can't mine per SOP" is true though. The only "lie" is the "produce materials" part. They have xenobio and xenoflora for it, and yes they can even go into space, hell they can even mine outside the colony as expedition prep exists for a reason (xenoarch but shh you can mine outside the colony, the only claim Lonestar have is to the underground caves)

Reply:
@ThePainkiller I'm not a part of discord nor server either, but i do like the build. Or i did, even, since the build has dropping its quality too fast and too harsh.
And both xenobio and xenoflora aren't effective, since you can harvest *some* considerable amount of resources like at 2h. At this point i can just ask somebody for making out and jerking off.

Reply:
@Lamasmaster 
> A plain lie, you can go to maintenence, to space to even just growing plants (the worst and least viable option)`

1)Maintenance isn't the most efficient or safe way of earning the resources, and some admins grumble about 'NO SCIENTISTS SHOULDNT DO THAT!!11!'. All other ways of gaining resources were stated above.

> the most expensive armors

2)Yet it has shitty stats. If the original armour was expensive af, the trouble isn't in protolathe, but in the original armour.

>you have laserproof armor which is now way better so enjoy that

3)It is shitty since only Soteria/Church works with weapons.

>but soteria really isn't going to be better at armor making than the literal blacksmith faction.

4)Then replace the removed content with something else, make Soteria good *at least at something*, ~~since LSS and Church both flex on its corpse, and now i get the reason why people want to remove Soteria~~

Reply:
Nobody cares if scientists loot maints or not, it's generally accepted everyone does it
The trouble is in the armor cause it costs even way less since the cost can be cut by upgrading parts, the original armor cannot
Lasers are good against mobs, not all of them but whatever you find in maints it's good.
Out of scope of this PR, besides soteria just got entire medical update and no talks of them ever being removed ever took place to my knowledge, blackshield prospectors lodge were only ever mentioned at any given points that they will be removed / reworked / merged.
It's part of the issue not the entire issue, ablatives, laserproof armor, mechs those are fine it's soteria niche to have anti energy weapon armors, not even moebius it's based on has this luxury, learn to accept boundaries of factions instead of but muh faction needs this cause this one has it.

Reply:
Or you can't just accept the fact that something is wrong and i have to point it out, g u h.

Reply:
I've seen multiple instances of admins actively either fucking with Scientists or admin messaging them to get out of maint/stop mining in xenoarch. Also, Soteria's armor in an apples to apples comparison is just straight up worse in every way to the Environment MK5. The only sense that it's comparable to it would be melee, and even then it's 5 lower. Jesus, their lives are already hard and tedious as it is. Only the trauma team has anything over it, that being the speed boost. The whole point of the armor Soteria can make is that it is cheap and shit when you compare it to everything that isn't a vest with metal sheets taped to it.